src/utils.js

```diff
@@ -27,11 +27,14 @@ const exec = async (command) => {
 
 const mimeType = async (opts) => {
   const { path, buffer } = opts;
-  FileMagic.magicFile = PATH.join(
-    __dirname,
-    '..',
-    'node_modules/@npcz/magic/dist/magic.mgc'
-  );
+  let magicFile = PATH.join(__dirname, '../assets/magic.mgc');
+  if (!fs.existsSync(magicFile))
+    magicFile = PATH.join(
+      __dirname,
+      '../node_modules/@npcz/magic/dist/magic.mgc'
+    );
+
+  FileMagic.magicFile = magicFile;
```

Comment:
If you upgrade to [`@npcz/magic@1.3.12`](https://github.com/npcz/magic/releases/tag/v1.3.12) this code can be simplified to the following:

```suggestion
  if (fs.existsSync(magicFile)) FileMagic.magicFile = magicFile; 
```

See https://github.com/npcz/magic/issues/10 for more information.

Reply:
https://github.com/iterative/cml/pull/901